pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p /tmp/logdir
+output_dir="/tmp/logdir"
+kubectl get po,svc,pvc --all-namespaces > $output_dir/kubectl-get-po-svc-pvc-all-namespace_output
+IFS=$'\n'
+for i in $(cat $output_dir/kubectl-get-po-svc-pvc-all-namespace_output | grep -i pravega)
+do
+echo $i
+if [[ $i == *"po"* ]];
+then 
+    if [ `echo $i | awk '{print $5}'` != 0 ]
```

Comment:
Be consistent with `;` before `then`.

My personal preference is including the `then` statement alongside with `if`.

```if [ condition ]; then```

Reply:
Yup. good point. I will update.